src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -44,29 +43,30 @@ public static OpenTelemetryLoggerOptions AddOtlpExporter(this OpenTelemetryLogge
         Action<OtlpExporterOptions> configure)
         => AddOtlpExporterInternal(loggerOptions, configure);
 
-    private static OpenTelemetryLoggerOptions AddOtlpExporterInternal(
-        OpenTelemetryLoggerOptions loggerOptions,
-        Action<OtlpExporterOptions> configure)
+    /// <summary>
+    /// Adds OTLP Exporter as a configuration to the OpenTelemetry ILoggingBuilder.
+    /// </summary>
+    /// <param name="loggerOptions"><see cref="OpenTelemetryLoggerOptions"/> options to use.</param>
+    /// <param name="configureExporterAndProcessor">Callback action for configuring <see cref="OtlpExporterOptions"/> and <see cref="LogRecordExportProcessorOptions"/>.</param>
+    /// <returns>The instance of <see cref="OpenTelemetryLoggerOptions"/> to chain the calls.</returns>
+    public static OpenTelemetryLoggerOptions AddOtlpExporter(
+        this OpenTelemetryLoggerOptions loggerOptions,
+        Action<OtlpExporterOptions, LogRecordExportProcessorOptions> configureExporterAndProcessor)
     {
         var exporterOptions = new OtlpExporterOptions();
+        var processorOptions = new LogRecordExportProcessorOptions();
 
-        // TODO: We are using span/activity batch environment variable keys
-        // here when we should be using the ones for logs.
-        var defaultBatchOptions = exporterOptions.BatchExportProcessorOptions;
-
-        Debug.Assert(defaultBatchOptions != null, "defaultBatchOptions was null");
-
-        configure?.Invoke(exporterOptions);
+        configureExporterAndProcessor?.Invoke(exporterOptions, processorOptions);
 
         var otlpExporter = new OtlpLogExporter(exporterOptions);
 
-        if (exporterOptions.ExportProcessorType == ExportProcessorType.Simple)
+        if (processorOptions.ExportProcessorType == ExportProcessorType.Simple)
```

Comment:
Not a blocker but you could probably update all the public extension methods to call `AddOtlpExporterInternal` to be consistent with Metrics: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs#L176-L194

Reply:
done